packages/launch/src/context.ts

```diff
@@ -16,6 +19,14 @@ const LaunchContext = React.createContext< LaunchContext >( {
 		// Won't work if trying to redirect the parent frame
 		window.location.href = url;
 	},
+	openCheckout: ( siteId, isEcommerce ) => {
+		window.top.location.href = addQueryArgs( `https://wordpress.com/checkout/${ siteId }`, {
```

Comment:
should it use the `redirectTo` function just defined above?

At the moment the default values of `redirectTo` and `openCheckout` don't align — one assumes (by default) that the Launch flow is in the context of Calypso, the other assumes that it's in the context of the iframed block editor

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/47601/commits/9b5e7daa93b6ceecdfb86e30f470db302b2d0543